pipeline_tools/input_utils.py

```diff
@@ -129,15 +146,17 @@ def get_smart_seq_2_fastq_names(metadata, version):
     Raises:
         NotImplementedError: if metadata version is unsupported
     """
-    if version.startswith('5.'):
-        return _get_smart_seq_2_fastq_names_v5(metadata)
-    elif version.startswith('4.'):
+    version_prefix = int(version.split('.', 1)[0])
+    print(version_prefix)
```

Comment:
nit: remove this print, or make it more informative :)

Reply:
Thanks! This was for debugging :)